CefSharp.MinimalExample.Wpf/Binding/Behaviors/BindHtmlAttachedBehavior.cs

```diff
@@ -40,8 +40,8 @@ private static void OnHtmlChanged(DependencyObject d, DependencyPropertyChangedE
             {
                 return;
             }
-
-            (d as ChromiumWebBrowser)?.LoadHtml((string)e.NewValue, "http://test/page");
+            
+            (d as LoadHtmlBehavior)?.AssociatedObject.LoadHtml((string)e.NewValue, "about:blank");
```

Comment:
I see...so `"http://test/page"` is a better fit? 

Should we add a new method to load an HTML directly through one parameter?

Could you explain me briefly the intent of the `LoadHtml` with an URL?